### PR TITLE
Selection: Add title and padding

### PR DESCRIPTION
**Pull request by mueller-ma**
Before:
screenshot_2017-10-04-20-39-30
After:
screenshot_2017-10-04-21-18-32

### CODE REVIEW

File: mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWidgetAdapter.java [hunk 1]
```diff
@@ -601,12 +599,13 @@ public void onItemClick(AdapterView<?> parent, View view, int index, long id) {
...
+					} catch (Exception ignored) {}
 				}
 			});
-    		spinnerAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
-    		selectionSpinner.setAdapter(spinnerAdapter);
-    		if (spinnerSelectedIndex >= 0) {
+			spinnerAdapter.setDropDownViewResource(R.layout.openhabwidgetlist_sectionswitchitem_spinner);
+			selectionSpinner.setAdapter(spinnerAdapter);
+			selectionSpinner.setPrompt(openHABWidget.getLabel());
+            if (spinnerSelectedIndex >= 0) {
```
Comment (FlorianSW) [inline mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWidgetAdapter.java]: you've mixed whitespaces with tabs :)

**Comment by mueller-ma** [comment]
I noticed that, but I couldnt find a way to use spaces in android studio, so I opened the file in vim and saw that the whole file is mixed.

File: mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWidgetAdapter.java (hunk 1 above)
Comment (mueller-ma) [reply mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWidgetAdapter.java]: I noticed that, but I couldnt find a way to use spaces in android studio, so I opened the file in vim and saw that the whole file is mixed.

Comment (mueller-ma) [reply mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWidgetAdapter.java]: We should write down rules for text indetion. Four spaces?

**Comment by digitaldan** [comment]
Due to the white space issue, the change set is not really readable :-) Can you point me to where in that file you changed/added?

**Comment by mueller-ma** [comment]
Just look at the first commit

**Comment by mueller-ma** [comment]
I have removed white space changes from this PR.

**Pull request merged**